modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -209,6 +209,33 @@ mirror:
     targetCatalog: my-namespace/my-operator-catalog
 ----
 
+// New example; multi-arch; all versions
+[discrete]
+[id="oc-mirror-image-set-examples-multi-arch_{context}"]
+== Use case: Including all versions of OpenShift Container Platform from a minimum to the latest
+
+The following `ImageSetConfiguration` file sets the `architectures` field to `multi` to include `multi` as the platform architecture which mirrors a multi-arch release image.
+
+Previously, oc-mirror users are able to mirror x86_64 release images when no platform architecture is specified in imageSetConfig. They could also mirror other architectures individually by adding other platform architectures in the imageSetConfig.
```

Comment:
Sorry, I missed it earlier, I think this should be placed in the beginning of the section, i.e before these lines:
>The following `ImageSetConfiguration` file sets the `architectures` field to `multi` to include `multi` as the platform architecture which mirrors a multi-arch release image.